tests/backwards_compatibility_tests/compatibility_test_runner.py

```diff
@@ -0,0 +1,603 @@
+import argparse
+import importlib
+import pkgutil
+import time
+
+import pytest
+from typing import Optional, Set
+import subprocess
+import sys
+import os
+import requests
+import semver
+import traceback
+
+from compatibility_test_logger import get_logger
+
+marqo_transfer_state_version = semver.VersionInfo.parse("2.9.0")
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
+from base_test_case import BaseCompatibilityTestCase
+
+# Keep track of containers that need cleanup
+containers_to_cleanup: Set[str] = set()
+volumes_to_cleanup: Set[str] = set()
+
+logger = get_logger(__name__)
+
+def load_all_subclasses(package_name):
```

Comment:
we need to do this so that we can iterate over all the classes that extend the baseCompatibilityTestCase class here: https://github.com/marqo-ai/marqo/pull/1024/files#diff-31e972f6614a426c5c437ec59d46adc623b3515dff2f90af7506afcec562333dR485-R493 and we can run prepare mode on these subclasses